core/client/app/services/config.js

```diff
@@ -2,27 +2,24 @@ import Ember from 'ember';
 
 const {Service, _ProxyMixin, computed} = Ember;
 
-function isNumeric(num) {
+function _isNumeric(num) {
```

Comment:
Agreed that it's already searchable but we've had a previous issue where the `isNumeric` method content was different and was changed to the current implementation which is why I feel the explicit naming is preferable - should probably have mentioned that in my last reply :smile: